docs/userGuide/syntax/pageLayouts.mbdf

```diff
@@ -114,27 +114,20 @@ which injects the actual page content in every page. This allows you to build la
 
 {{ icon_example }} Adding statistics formula to the bottom of every page
 
-<box>
-<i>index.md</i>
-<br><br>
-
-  ```html
+  ```html {heading="index.md"}
   #### Main content of your page
 
   Content of your page
   ```
-</box>
 
-<box>
-<i>page.md</i>
-<br><br>
-  {{ mainContentBody }}
-  ```html
+  ```html {heading="page.md"}
+    <variable name="mainContentBody">
```

Comment:
No, I didn't mean to insert this here 😅 If you look at the top of this file, there is a variable declared for `mainContentBody`, and the value is substituted whenever we use the variable by writing `{{ mainContentBody }}`. We should change the value of the variable, and continue to use the variable here.

Reply:
Aah sorry. Didn't realise there was a variable declared on top of the file. Thanks for guiding.

Also, can you help explain what does `&#8203` do?

Reply:
It's the unicode for zero width space. This allows us to break up the `{{ variable }}` syntax and avoid markbind from treating it as a variable.

Let's also leave a comment explaining why we are doing this.

Reply:
Please also leave a TODO to remind us to change to a proper escape tag once #1049 is ready.

Reply:
Done!